utils/doclint/preprocessor/test.js

```diff
@@ -156,6 +156,42 @@ describe('runCommands', function() {
         #### page.$
         #### page.$$`);
     });
+    it('should work with code blocks', () => {
+      const source = new Source('doc.md', `<!-- gen:toc -->XXX<!-- gen:stop -->
+        ### class: page
```

Comment:
nit: remove tabs here and below.

Reply:
that's aligned with other tests :-) if you really want, I'll follow up for them all!